atd-vzd/schema/ems__incidents.sql

```diff
@@ -0,0 +1,147 @@
+DROP TABLE public.ems__incidents;
+
+CREATE TABLE public.ems__incidents (
+	id serial primary key,
+	pcr_key int4 NOT NULL,
+	crash_id integer default null,
+	incident_date_received date NULL,
+	incident_time_received time NULL,
+	incident_number text NULL,
+	incident_location_address text NULL,
+	incident_location_city text NULL,
+	incident_location_state text NULL,
+	incident_location_zip text NULL,
+	incident_location_longitude float NULL,
+	incident_location_latitude float NULL,
+	incident_problem text NULL,
+	incident_priority_number text NULL,
+	pcr_cause_of_injury text NULL,
+	pcr_patient_complaints text NULL,
+	pcr_provider_impression_primary text NULL,
+	pcr_provider_impression_secondary text NULL,
+	pcr_outcome text NULL,
+	pcr_transport_destination text NULL,
+	pcr_patient_acuity_level text NULL,
+	pcr_patient_acuity_level_reason text NULL,
+	pcr_patient_age integer NULL,
+	pcr_patient_gender text NULL,
+	pcr_patient_race text NULL,
+	mvc_form_airbag_deployment text NULL,
+	mvc_form_airbag_deployment_status text NULL,
+	mvc_form_collision_indicators text NULL,
+	mvc_form_damage_location text NULL,
+	mvc_form_estimated_speed_kph integer NULL,
+	mvc_form_estimated_speed_mph integer NULL,
+	mvc_form_extrication_comments text NULL,
+	mvc_form_extrication_datetime timestamp NULL,
+	mvc_form_extrication_required_flag integer NULL,
+	mvc_form_patient_injured_flag integer NULL,
+	mvc_form_position_in_vehicle text NULL,
+	mvc_form_safety_devices text NULL,
+	mvc_form_seat_row_number text NULL,
+	mvc_form_vehicle_type text NULL,
+	mvc_form_weather text NULL,
+	pcr_additional_agencies text NULL,
+	pcr_transport_priority text NULL,
+	pcr_patient_acuity_initial text NULL,
+	pcr_patient_acuity_final text NULL,
+	unparsed_apd_incident_numbers text NULL,
+	apd_incident_numbers integer[] NULL,
```

Comment:
We can get 0, 1, or more incident numbers here, so we're storing them in an array. Also keeping the unparsed value, just in case.